frontend/README.md

```diff
@@ -0,0 +1,151 @@
+# Translate.Next — a better Translate app for Pontoon
+
+
+## Tools
+
+<table>
+    <tr>
+        <td>Bootstrapper</td>
+        <td>[create-react-app](https://github.com/facebook/create-react-app)</td>
```

Comment:
These links don't seem to render:
https://github.com/adngdb/pontoon/blob/translate.next/frontend/README.md